jsonapi/jsonapi_utility.go

```diff
@@ -26,6 +29,14 @@ const (
 	ErrorCodeDataConflict      = "data_conflict_error"
 )
 
+var mutex sync.Mutex
+
+func init() {
+	c, err := configuration.Get()
+	_ = err // Log this
```

Comment:
Need help on this, not sure how to handle this properly

Reply:
If I understand sentry docs right you do not have to set DSN explicitly at all because SENTRY_DSN is available (it's set in openshift/core.app.yaml). So, just remove this init() block and all your changes in the configuration. It should work.

Reply:
Yeah, this not technically needed at this point

Reply:
done!